docs/user-guide/migrating-from-jscs.md

```diff
@@ -0,0 +1,79 @@
+# Migrating from JSCS
+
+In April 2016, we [announced](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint) that the JSCS project was shutting down and the JSCS team would be joining the ESLint team. This guide is intended to help those who are using JSCS to migrate their settings and projects to use ESLint. We've tried to automate as much of the conversion as possible, but there are some manual changes that are needed.
+
+## Terminology
```

Comment:
I think jscodeshift does a pretty decent job at preserving the original file. Worst case scenario, we could just warn the users that the transformation cannot be 100% trusted and that they should double-check the diff manually after the transform runs. I'm totally willing to take lead on this, perhaps we could at least experiment with an extended POC like the one @hzoo posted above!
Let me know what you guys think :)